### ISSUE TITLE
Add preferredLanguage to Person

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1084]
Adding an issue to match @vholland's concrete implementation/proposal #291 

Use case is describing (in broad terms) a person's preferred language. There is some detailed discussion in #291 but rough consensus that doing something here makes sense. Feel's like we're nearly there - just needs another push on the detail.

**Comment by danbri** [comment]
Here is a summary of the comments in the pull request #291:

* @chaals: "Seems reasonable, modulo the fact that many people have more than one preferred languages"
* @vholland: "I am assuming people can set more than one language"
* @chaals: "OK. Works for Me. I support adding this."
* @thadguidry: "+1 for preferredLanguage"
* @danbri: "+1 on adding this"
* @sesuncedu points out that Language is "is defined as a union of two
(almost disjoint) classes" which makes using it as a range awkward here. "also a case to be made for having required languages be usable on
job postings" -> new issue?
* @tmarshbing: "I also support adding this, but I would like to see a little more discussion before we do", "would also like to see some language property on Country." (-> new issue?)
* @inkel: "I think preferredLanguage is better than nothing, but I also think that it falls short on its premise, even when more than one could be indicated. Again, is not an easy topic." "I wouldn't go into a more detailed description like preferredVerbalLanguage or preferredWrittenLanguage, I'd start with something simpler."
* @guha: "Something like this would be very useful, but I agree that it needs more discussion. I'd like to see a set of examples and use cases before we can decide."

**Comment by danbri** [comment]
As one example for where this might be useful (more welcomed):

* FoodEstablishmentReservation can be used to describe details of a reservation, including a person via underName. Subsequent interactions e.g. in emails would benefit from information to guide choice of language (and locale).

**Comment by thadguidry** [comment]
I also want to make the definition a bit more clear to avoid confusion.

The intent (the definition) ideally should be expanded to include something
like "The preferred language(s) from the many that someone or something
might know how to speak, read, or understand, for example the Pope prefers
speaking in his native Spanish and Italian, over his ability to speak the
languages of Spanish, Italian, Portuguese, French, German, Ukrainian and
Piedmontese (a language spoken in the Piedmont, an area in northern Italy).

Also a relevant Action property would be under
https://schema.org/CommunicateAction to narrow the choices contained in the
inLanguage.
